docs/integrations/dbt.md

```diff
@@ -214,6 +214,25 @@ seeds:
         data_type: <SQL data type>
 ```
 
+Alternatively, you can specify a [column_types](https://docs.getdbt.com/reference/resource-configs/column_types) dictionary in your `dbt_project.yml` file, where the keys define the column names and the values the data types.
+
+``` yaml
+seeds:
+  <seed name>
+    +column_types:
+        <column name> : <SQL data type>
```

Comment:
[Nit] let's be consistent and use 2 space indentation and no space between `<column name>` and `:` (ditto for the section below).